packages/components/src/components/dropdown/dropdown.tsx

```diff
@@ -201,33 +259,28 @@ const Dropdown = ({
     suffix_icon,
     test_id,
     value,
-}) => {
-    const dropdown_ref = React.useRef();
-    const native_select_ref = React.useRef();
-    const wrapper_ref = React.useRef();
+}: TDropdown) => {
+    const dropdown_ref = React.useRef() as React.MutableRefObject<HTMLDivElement>;
+    const native_select_ref = React.useRef() as React.MutableRefObject<HTMLDivElement>;
+    const wrapper_ref = React.useRef() as React.MutableRefObject<HTMLDivElement>;
     const nodes = React.useRef(new Map());
-    const list_ref = React.useRef();
+    const list_ref = React.useRef() as React.MutableRefObject<HTMLDivElement>;
     const is_portal = !!list_portal_id;
 
     const [is_list_visible, setIsListVisible] = React.useState(!!is_nativepicker_visible);
     const initial_render = React.useRef(true);
-    useBlockScroll(list_portal_id && is_list_visible ? dropdown_ref : false);
+    useBlockScroll(list_portal_id && is_list_visible && dropdown_ref);
```

Comment:
We must pass `RefObject<HTMLElement>` to `useBlockScroll` and it doesn't take **undefined** or **false** or anything else, _I changed this but still I have ts error here because we may pass undefined to this custom hook!_
I would be happy if there is any suggestion for fixing this :pray: